lib/pub_sub/message.rb

```diff
@@ -1,7 +1,7 @@
 module PubSub
   class Message
     def initialize(payload)
-      @payload = JSON.parse(payload)
+      @payload = JSON.parse(JSON.parse(payload)['Message'])
```

Comment:
Perhaps we can ask for the mode and then change the behaviour here to act accordingly